### ISSUE TITLE
[Disable grouping on taskbar] Folder icon shown on pinned items

### ISSUE THREAD

**Issue by theFireProject** [ramensoftware/windhawk-mods#1153]
The problem described in issue #644 still exists on my second monitor. After closing Windows Explorer, the pinned item still shows the window icon (which is the folder icon of the current directory) instead of the explorer.exe icon:

**Screenshots**

<img src="https://github.com/user-attachments/assets/5c153e1f-4499-4bb4-95de-5dd49b31482f" width="40%">

<img src="https://github.com/user-attachments/assets/b3392aa4-758a-48bf-b36e-90831d54ff46" width="40%">

**Comment by m417z** [comment]
Yes, it was discovered here: https://github.com/ramensoftware/windhawk-mods/issues/462#issuecomment-2427875175.
It will be fixed in the next version.

Meanwhile, if you want to try the fix right away, you can try this version with the fix:
https://github.com/m417z/my-windhawk-mods/blob/0df5c95802e60dea7afa61b7fc41158e2d8de344/mods/taskbar-grouping.wh.cpp

To try it, disable the existing mod, then click the bottom right button to create a new mod, replace the code with the test mod code, click "Compile Mod", then "Exit Editing Mode".

**Comment by m417z** [comment]
Fixed in version 1.3.8.
